crates/egui_demo_lib/src/demo/font_book.rs

```diff
@@ -79,7 +79,11 @@ impl crate::View for FontBook {
                 ui.spacing_mut().item_spacing = egui::Vec2::splat(2.0);
 
                 for (&chr, name) in named_chars {
-                    if filter.is_empty() || name.contains(filter) || *filter == chr.to_string() {
+                    if filter.is_empty()
+                        || name.contains(filter)
+                        || format!("{:x}", chr as u32).starts_with(filter)
+                        || filter.contains(chr)
```

Comment:
`*filter == chr.to_string()` is for entering a specific glyph (e.g. `Å`) to find it.

`filter.contains(chr)` should work for that too, but entering "infiity" will now include all the letters that make up the word "infinity", which seems a bit much?

Maybe it would make sense to pull this out to its own function with a few small unit tests

Reply:
> `*filter == chr.to_string()` is for entering a specific glyph (e.g. `Å`) to find it.
> 
> `filter.contains(chr)` should work for that too, but entering "infiity" will now include all the letters that make up the word "infinity", which seems a bit much?
> 
> Maybe it would make sense to pull this out to its own function with a few small unit tests

It has the same effect as `*filter == chr.to_string()`.

When you type "infiity", the intention is to include all the letters that make up the word "infinity". 
That was convenient to use.